src/client/src/rt-theme/ThemeState.tsx

```diff
@@ -19,14 +19,14 @@ interface ThemeSelector {
 }
 
 interface ThemeStateProps extends ThemeSelector {
-  onChange?: (value: ThemeSelector) => void
+  onChange?: (value: string | null) => void
 }
 
 export interface ThemeStateValue extends ThemeSelector {
   setTheme: (options: ThemeSelector) => void
 }
 
-const { Provider: ContextProvider, Consumer: ContextConsumer } = React.createContext(null)
+const { Provider: ContextProvider, Consumer: ContextConsumer } = React.createContext<ThemeStateValue | null>(null)
```

Comment:
Instead of null, just pass in `{setTheme: ()=>{}}`

Reply:
Nice one. Wil do

Reply:
I thought about that. It would have to be

```{name:null, setTheme: ()=>{}}```

Reply:
Actually I will do that as I can remove a level of nulls. Might need to update some of the guards though